core/tests/performance_tests/string_classifier_performance_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Performance tests for string classifier.
+
+Run this script from the Oppia root directory:
+
+    export PYTHONPATH=$PYTHONPATH:.
+    python core/tests/performance_tests/string_classifier_performance_test.py
+
+"""
+
+import os
+import time
+import yaml
+
+from core.domain.classifier_services import StringClassifier
+
+def benchmark(func):
+    def time_taken(obj, num):
+        start = time.time()
+        result = func(obj, num)
+        end = time.time()
+        print '%s spent %f seconds for %d instances' % (func.__name__,
+                                                        end - start, num)
+        return result
+    return time_taken
+
+class StringClassifierPerformanceTest(object):
```

Comment:
Please add docstring explaining, in some detail, what this class represents and how to use it.